### PR TITLE
Update GCMR Assembly Model Description

### LINKED ISSUE #309: GCMR missing certain dimensions on Model Description Page
## Description
I am working on modeling the VTB GCMR, and I noticed the model [description](https://mooseframework.inl.gov/virtual_test_bed/microreactors/gcmr/GCMR_Assembly_Model_Description.html) was missing information about the moderator compact radius and thickness of the FeCrAl envelope. I was able to get it after doing some investigating, but I believe these details should be added to the linked page. This isn't exactly a "bug" but bug report was the closest option in the issue templates.

## Impact
This slows down others who would want to model this system, as they'd have to do some digging when it would be fairly easy to add these details to the page.

## Further on
Another small nitpick would be that it would be useful to provide more info about the TRISO and B4C particles on this page, but that one is more up for debate if it is worth including. Could argue people should choose from the literature for their model, though I'd argue having a centralized list of radii/materials is better for cross-code comparison.

### PR DESCRIPTION
**Pull request by lewisgross1296**
Closes  #309

### CODE REVIEW

**Comment by GiudGiud** [comment]
@nstauff or @aabdelhameedd for signing off on the modification

File: doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md [hunk 1]
```diff
@@ -27,6 +27,27 @@ The major technical parameters for the GC-MR assembly model are:
...
+Both the TRISO and B4C poison particles are composed of concentric spherical regions. The particles are packed into graphite compacts. The TRISO particles have the following regions
+
+| Region | Outer Radius (cm) | Density ($g/cm^{3}$) |
+| - | - | - |
+| UCO | 212.5E-4 | 10.5 |
+| C (buffer) | 312.5E-4 | 1.0 |
+| PyC1 | 352.5E-4 | 1.9 |
+| SiC | 387.5E-4 | 3.2 |
+| PyC2 | 427.5E-4 | 1.87 |
+
```
Comment (nstauff) [inline doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md:46]: densities need updating:
UCO: 10.744
C(buffer): 1.04 (TRISO and BP)
PyC: 1.882 (TRISO and BP)
SiC: 3.171  (TRISO and BP)
B4C: 2.47

Comment (lewisgross1296) [reply doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md:46]: Glad I posted those to check! I'll fix and push it

File: doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md [hunk 2]
```diff
@@ -27,6 +27,27 @@ The major technical parameters for the GC-MR assembly model are:
 | Total height (cm) | 200 |
 | Active height (cm) | 160 |
 | TRISO fuel compact radius (cm) | 0.9 |
-| Burnable poison radius (cm) | 0.25 |
-| Coolant hole radius (cm) | 0.6 |
-| Control hole radius (cm) | 0.99 |
+| Moderator compact radius (cm), density ($g/cm^{3}$) | 0.843, 4.2 |
```
Comment (nstauff) [inline doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md:30]: moderator density: 4.085

**Review by nstauff — CHANGES_REQUESTED** [review]

File: doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md (hunk 1 above)
Comment (nstauff) [reply doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md:46]: I confirm I used the same density for this model.

File: doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md [hunk 3]
```diff
@@ -27,6 +27,27 @@ The major technical parameters for the GC-MR assembly model are:
...
+| PyC1 | 352.5E-4 | 1.882 |
+| SiC | 387.5E-4 | 3.171 |
+| PyC2 | 427.5E-4 | 1.882 |
+
+The B4C poison particles have the following regions
+
+| Region | Outer Radius (cm) | Density ($g/cm^{3}$) |
+| - | - | - |
+| B4C | 100.0E-4  | 2.47 |
+| C (buffer) | 118.0E-4 | 1.0 |
```
Comment (nstauff) [inline doc/content/microreactors/gcmr/GCMR_Assembly_Model_Description.md:52]: still 1.04 on the C-buffer

**Review by nstauff — CHANGES_REQUESTED** [review]

**Review by nstauff — APPROVED** [review]

**Pull request merged**